Lib/tracemalloc.py

```diff
@@ -209,11 +209,14 @@ def __str__(self):
     def __repr__(self):
         return "<Traceback %r>" % (tuple(self),)
 
-    def format(self, limit=None):
+    def format(self, limit=None, reverse=False):
         lines = []
         if limit is not None and limit < 0:
             return lines
-        for frame in self[:limit]:
+        frame_slice = self[:limit]
```

Comment:
Hum, I'm not sure that it's useful to truncate before reversing the frames. I would suggest to truncate after reversing.

Reply:
The rationale was that the function is designed to format the frames of the `limit` last calls. If truncating after reversing, it will format the first `limit` calls, which I think would be less intuitive and less useful.

Reply:
My bad, you are right :-)